### ISSUE TITLE
Command doesn't exit if thread is stuck in blocking I/O

### ISSUE THREAD

**Issue by jethrogb** [fortanix/rust-sgx#109]
This program may not terminate:

```rust
use std::{net::TcpListener, sync::mpsc, thread};

fn main() {
    let (tx, rx) = mpsc::channel();
    thread::spawn(move || {
        let listen = TcpListener::bind("0:0").unwrap();
        tx.send(()).unwrap();
        while let Ok(_) = listen.accept() {}
    });
    rx.recv().unwrap();
    println!("Exiting main thread");
}
```

This is because when the main thread exits, the second thread is likely to be blocking in an `accept()` call. Therefore, that thread never terminates and the main thread (out of the enclave) is waiting indefinitely for that second thread to finish.

**Comment by jethrogb** [comment]
I expect this to be fixed alongside #18.

**Comment by jethrogb** [comment]
Closing since #18 is merged. Ok, @mzohreva ?

**Comment by mzohreva** [comment]
Yeah, this was fixed in #223
